fth/file.fth

```diff
@@ -50,6 +50,8 @@ private{
 create (LINE-TERMINATOR) \n c,
 : LINE-TERMINATOR ( -- c-addr u ) (line-terminator) 1 ;
 
+-72 constant THROW_RENAME_FILE
```

Comment:
Yes, it's part of the standard, cf. this [table](http://lars.nocrew.org/forth2012/exception.html#table:throw).
Some of the codes are defined in pf_io.h.  It's a disadvantage of defining RENAME-FILE in Forth that those constants aren't easily accessible.